lib/data/entity.js

```diff
@@ -42,9 +42,14 @@ const validateEntity = (entity) => {
   if (matches == null) throw Problem.user.entityViolation({ reason: `ID [${id}] is not a valid UUID.` });
   const uuid = matches[2];
 
+  // If create is not an allowed value of true, don't return entity data
+  if (entity.system.create !== '1' || entity.system.create === 'true')
```

Comment:
Shouldn't this be either `entity.system.create !== '1' && entity.system.create !== 'true'` or `!(entity.system.create === '1' || entity.system.create === 'true')`